editor/icons/README.md

```diff
@@ -2,7 +2,7 @@ The icons here are optimized SVGs, because the editor renders the svgs at runtim
 to be small in size, so they can be efficiently parsed.
 
 The original icons can be found at:
-https://github.com/djrm/godot-design/tree/master/assets/icons
+https://github.com/djrm/godot-design/tree/master/engine/icons
```

Comment:
Actually it should be https://github.com/godotengine/godot-design/tree/master/engine

Reply:
Thanks Akien, updated it now